src/perform_step/exponential_rk_perform_step.jl

```diff
@@ -576,6 +576,95 @@ function perform_step!(integrator, cache::HochOst4Cache, repeat_step=false)
   # integrator.k is automatically set due to aliasing
 end
 
+#############################################
+# EPIRK integrators
+function perform_step!(integrator, cache::Exp4ConstantCache, repeat_step=false)
+  @unpack t,dt,uprev,f,p = integrator
+  A = f.jac(uprev, p, t)
+  alg = typeof(integrator.alg) <: CompositeAlgorithm ? integrator.alg.algs[integrator.cache.current] : integrator.alg
+  f0 = integrator.fsalfirst # f(uprev) is fsaled
+  ts = [dt/3, 2dt/3, dt]
+  kwargs = [(:tol, integrator.opts.reltol), (:iop, alg.iop), (:norm, integrator.opts.internalnorm), (:adaptive, true)]
+
+  # Krylov for f(uprev)
+  B1 = [zeros(f0) f0]
+  K1 = phiv_timestep(ts, A, B1; kwargs...) # tϕ(tA)f0
+  @inbounds for i = 1:3
+    K1[:,i] ./= ts[i]
+  end
+  w4 = K1 * [-7/300, 97/150, -37/300]
+  u4 = uprev + dt * w4
+  d4 = f(u4, p, t+dt) - f0 - dt * (A*w4) # TODO: what should be the time?
```

Comment:
If you sum together the row of a_ij coefficients you should get the c_i coefficients for the time. However, there's no guarantee that this is higher order though since there would be extra order conditions which are dropped in the derivation for autonomous systems. Thus it's probably best to just leave it like this, and document this as first-order for non-autonomous systems.